mitodl/hugo-course-publisher
# full color home page

#### Pre-Flight checklist


#### What are the relevant tickets?

no github issue, design here: https://projects.invisionapp.com/d/main#/console/18609637/407710624/inspect

#### What's this PR do?

this implements the "full color" design for the homepage, especially restyling the top bar.

#### How should this be manually tested?

I think just check everything out and make sure I copied the styles over correctly from the design and that it looks and feels good and all that.


#### Screenshots (if appropriate)

![Screenshot from 2020-03-30 10-59-53](https://user-images.githubusercontent.com/6207644/77927753-94b74400-7275-11ea-880a-c3beedf8146a.png)
![Screenshot from 2020-03-30 10-59-47](https://user-images.githubusercontent.com/6207644/77927755-94b74400-7275-11ea-93c5-2bf1bf59e7fe.png)
![Screenshot from 2020-03-30 10-57-33](https://user-images.githubusercontent.com/6207644/77927756-954fda80-7275-11ea-8055-d747e353cdeb.png)
